pkg/controller/registry/resolver/source_registry.go

```diff
@@ -109,7 +109,7 @@ func (s *registrySource) Snapshot(ctx context.Context) (*cache.Snapshot, error)
 	// Fetching default channels this way makes many round trips
 	// -- may need to either add a new API to fetch all at once,
 	// or embed the information into Bundle.
-	defaultChannels := make(map[string]string)
+	packages := make(map[string]*api.Package)
 
```

Comment:
So these changes to the resolver packages are still required to make the deprecation information available in the entry, right?

Reply:
That's right, these are still needed.